app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -179,6 +180,25 @@ void openPdf() {
         }
     }
 
+    /**
+     * Opens ImagePickerActivity to select Images
+     */
+    private void selectImages() {
+        Intent intent = new Intent(mActivity, ImagePickerActivity.class);
+
```

Comment:
you have even removed the functionalities from this function

Reply:
I only added mCreatePdf.setEnabled(true); at line number 199

Reply:
look at the file changed in this PR. If I merge your PR, all this will be changed. So, it's a problem, because you have other changes too.

Reply:
OK!